api/HTMLHyperlinkElementUtils.json

```diff
@@ -0,0 +1,750 @@
+            },
+            "ie": {
+              "version_added": false,
+              "notes": "Though not grouped in a single abstract interface, these methods are directly available on the interfaces that implement it, if this interface is supported."
+            },
+            "opera": {
+              "version_added": false,
+              "notes": "Though not grouped in a single abstract interface, these methods are directly available on the interfaces that implement it, if this interface is supported."
+            },
+            "opera_android": {
+              "version_added": false,
+              "notes": "Though not grouped in a single abstract interface, these methods are directly available on the interfaces that implement it, if this interface is supported."
+            },
+            "safari": {
+              "version_added": false,
+              "notes": "Though not grouped in a single abstract interface, these methods are directly available on the interfaces that implement it, if this interface is supported."
+            },
+            "safari_ios": {
+              "version_added": false,
+              "notes": "Though not grouped in a single abstract interface, these methods are directly available on the interfaces that implement it, if this interface is supported."
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "username": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLHyperlinkElementUtils/username",
+          "support": {
+            "webview_android": {
+              "version_added": true,
+              "notes": "Starting in Chrome 52, this property was moved to <a href='https://developer.mozilla.org/docs/Web/API/URL'>URL</a>"
+            },
+            "chrome": {
+              "version_added": true,
+              "notes": "Starting in Chrome 52, this property was moved to <a href='https://developer.mozilla.org/docs/Web/API/URL'>URL</a>"
+            },
+            "chrome_android": {
+              "version_added": true,
+              "notes": "Starting in Chrome 52, this property was moved to <a href='https://developer.mozilla.org/docs/Web/API/URL'>URL</a>"
+            },
+            "firefox": {
+              "version_added": "26",
+              "notes": "From Gecko 26 to Gecko 44, this property was on the <code>URLUtils</code> mixin. It has been moved either on the <code>HTMLHyperlinkElementUtils</code> mixin, or directly on the interface."
+            },
+            "firefox_android": {
+              "version_added": "26",
+              "notes": "From Gecko 26 to Gecko 44, this property was on the <code>URLUtils</code> mixin. It has been moved either on the <code>HTMLHyperlinkElementUtils</code> mixin, or directly on the interface."
```

Comment:
It's been decided not to mention Gecko. Please change "From Gecko 22 to Gecko 44..." to From Firefox 22 to Firefox 44..."